cmd.md

```diff
@@ -0,0 +1,94 @@
+# Runtime Command Line Interface
+
+This document specifies the command line syntax for runtime operations defined in the [Runtime and Lifecycle](runtime.md) specification.
+It is STRONGLY RECOMMENDED that implementations adhere to the command line definition defined below to ensure interoperability.
+This document will not duplicate the information/semantics specified in the [Runtime and Lifecycle](runtime.md) document, rather it just focuses on syntax of the command line.
+
+## Convention
+
+For the purpose of this document, the following syntactical conventions are used:
+
+* `[ ... ]` denotes an optional field
+* `< ... >` denotes a substitution field. 
+The word(s), or phrase, within the `<>` describes the information to be passed-in.
+
+## General Format and Behavior
+
+The general format of all commands MUST be:
+```
+runtime [global-options] action [action-specific-options] [arguments]
+```
+
+Upon successful running of an action, the exit code MUST be zero.
+
+If there is an error during the running of an action, then:
+* the exit code MUST be non-zero
+* any error text MUST be displayed on stderr
+* the state of the environment MUST be the same as if the action was never attempted (modulo any possible trivial ancillary changes such as logging)
```

Comment:
I think this last line is an unachievable goal, and since #418 the spec has cut runtimes a bit more slack.  Attempting cleanup (as required by #418) doesn't mean “successfully cleaned up” (see [here][1]).

[1]: https://github.com/opencontainers/runtime-spec/blame/v1.0.0-rc1/runtime.md#L124